package.json

```diff
@@ -28,12 +28,13 @@
         "cross-env": "7.0.3",
         "esbuild": "0.16.10",
         "esbuild-gas-plugin": "0.5.0",
-        "eslint": "8.30.0",
+        "eslint": "^8.31.0",
```

Comment:
I am trying to stick to exact versions to be sure the build is on exact versions that I checked are good.
Am I wrong? Maybe I don't remember exactly how it works